src/Microsoft.Azure.ServiceBus/ExceptionReceivedEventArgsAction.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Microsoft.Azure.ServiceBus
+{
+    using Microsoft.Azure.ServiceBus.Primitives;
+
+    /// <summary>Action taking place when <see cref="ExceptionReceivedEventArgs"/> is raised for a processed message.</summary>
```

Comment:
btw, I tweaked it a little. Will let the team come up with a better xml summaries.